base_layer/core/src/transactions/transaction_components/test.rs

```diff
@@ -137,12 +137,12 @@ fn range_proof_verification() {
         },
     }
 
-    let value = 2u64.pow(32) + 1;
+    let value = 2u64.pow(32);
```

Comment:
This change looks a bit sus

Reply:
Ok I will fix that test. It will fail with `construct_proof` if the value is too large.